treescript/src/treescript/github/versionmanip.py

```diff
@@ -23,8 +23,7 @@
 _VERSION_CLASS_PER_BEGINNING_OF_PATH = {}
 
 _VERSION_CLASS_PER_END_OF_SOURCE_REPO = {
-    # previously, this mapped mobile github repos to the MobileVersion; now there are no remaining
-    # repos on github requiring version bumps
+    "firefox-ios": MobileVersion,
```

Comment:
iOS doesn't use the same version scheme as other mobile projects.

Instead of bumping `139.0` -> `139.0.1`, they go `139.0` -> `139.1`. Need to figure out if mozilla-versions has a scheme that works or if we need to create one

Reply:
https://github.com/mozilla-releng/mozilla-version/pull/134